src/test/kotlin/com/intellij/ml/llm/template/BatchProcessingExtractFunction.kt

```diff
@@ -0,0 +1,195 @@
+package com.intellij.ml.llm.template
+
+import com.intellij.ml.llm.template.models.GPTExtractFunctionRequestProvider
+import com.intellij.ml.llm.template.models.LLMRequestProvider
+import com.intellij.ml.llm.template.utils.*
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiNameIdentifierOwner
+import com.intellij.psi.util.PsiTreeUtil
+import com.intellij.testFramework.LightPlatformCodeInsightTestCase
+import com.mongodb.client.MongoClient
+import com.mongodb.client.MongoClients
+import org.bson.*
+import org.junit.jupiter.api.TestInstance
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Could you please also add a guide on how to run it locally? I mean what you have shown our on the last meeting, collecting and processing samples and so on

Reply:
Sure, I'll will add a "how-to" section

Reply:
Like I have mentioned above, I will remove this code for now, but include it in a future PR as a separate module